easybuild/easyblocks/v/vep.py

```diff
@@ -97,8 +97,8 @@ def install_step(self):
             # l: Bio::DB::HTS, should be provided via EasyBuild
             # p: plugins
             '--AUTO af',
-            # install all species
-            '--SPECIES all',
+            # install selected species
+            '--SPECIES cyprinus_carpio_carpio',
```

Comment:
What about using a variable and allowing the installer to decide what to include via the easyconfig? See https://github.com/easybuilders/easybuild-easyblocks/blob/develop/easybuild/easyblocks/generic/conda.py#L45-L54 for an example of how to do that. If you call the variable `species` then this line would become:
```
'--SPECIES %s' % self.cfg['species'],
```